ibm/mas_devops/roles/suite_manage_doclinks_config/README.md

```diff
@@ -7,7 +7,7 @@ You can run `cos` role to provision an IBM Cloud Object Storage or you can provi
 
```

Comment:
Can we state in readme that the old role will be deprecated and that to switch providers, ManageWorkspace CR / serverBundleProperties needs to be clean up?

Reply:
Changes added.